ert/data/record/_record.py

```diff
@@ -236,36 +238,43 @@ def collection_type(self) -> RecordCollectionType:
         return self._collection_type
 
 
-def path_to_bytes(file_path: pathlib.Path) -> bytes:
+async def path_to_bytes(file_path: pathlib.Path) -> bytes:
```

Comment:
This function has a very specific goal: handling a tar file, so maybe it should be renamed. Also, maybe it should live in some utilities file?

Reply:
Agree. `make_tar`?